backend/tracim_backend/lib/core/system.py

```diff
@@ -85,7 +88,7 @@ def get_config(self) -> ConfigModel:
             call__enabled=self._config.CALL__ENABLED,
             call__unanswered_timeout=self._config.CALL__UNANSWERED_TIMEOUT,
             auth_types=auth_types,
-            idp_list=self._config.IDP_LIST,
+            idp_list=idp_list,
```

Comment:
why not a lambda for a more explicit/direct code read?

```
idp_list=[idp.to_dict() for idp in self._config.IDP_LIST]
```

What is the need for an intermediate variable? debugging?

Reply:
IIRC this didn't work for some reason, will try again as you proposed to be sure